src/iqm/iqm_client/iqm_client.py

```diff
@@ -888,24 +888,25 @@ def submit_circuits(
         except (json.decoder.JSONDecodeError, KeyError) as e:
             raise CircuitExecutionError(f'Invalid response: {result.text}, {e}') from e
 
-    def get_run(self, job_id: UUID) -> RunResult:
+    def get_run(self, job_id: UUID, timeout_secs: float = REQUESTS_TIMEOUT) -> RunResult:
```

Comment:
```suggestion
    def get_run(self, job_id: UUID, *, timeout_secs: float = REQUESTS_TIMEOUT) -> RunResult:
```
And the same elsewhere. Keyword only since this is a parameter the user should only rarely need to set.

Reply:
Implemented the suggestion